packages/buefy/src/components/autocomplete/Autocomplete.vue

```diff
@@ -35,7 +35,10 @@
                 class="dropdown-menu"
                 :class="{ 'is-opened-top': isOpenedTop && !appendToBody }"
                 :style="style"
-                v-show="isActive && (!isEmpty || hasEmptySlot || hasHeaderSlot || hasFooterSlot)"
+                v-show="
```

Comment:
unusual formatting ...I found it more intuitive on a single line ...just nitpicking here, you can ignore this if you want to